### ISSUE TITLE
Issues with viewing all bans

### ISSUE THREAD

**Issue by poixen** [woogerboy21/servatriceadminwebui#35]
Viewing all bans is slow and can crash the browser (chrome)

**Comment by woogerboy21** [comment]
yeah, it was an issue in the past that was identified. when the ban table grows so large the page should only display X number of bans with multiple page support.

**Comment by woogerboy21** [comment]
Oh and btw @poixen , my firefox doesnt crash..... must be that new fangled software browser your using :)
